ogm/releases/5.3/index.adoc

```diff
@@ -0,0 +1,36 @@
+:awestruct-layout: project-releases-series
+:awestruct-project: ogm
+:awestruct-series_version: "5.3"
+
+=== Core
+
+==== Hibernate ORM 5.2 support
+
+As of Hibernate OGM 5.2, the version of ORM to use with OGM is 5.2 (use the latest micro version).
+
+==== Hibernate Search 5.9 support
+
+As of Hibernate Search 5.9, the version of Search to use with Search is 5.3 (use the latest micro version).
+
+==== Upgrade HQL Parser to 1.5.0.Final
+
+Hibernate Query Parser 1.5.0.Final is now supported.
+
+=== Datastores
+
+==== Infinispan
+
+ * Infinspan Clustered Counters for sequences generation with Infinispan Embedded are now created at startup
+ * Infinspan Clustered Counters name for id generation also contain the table and not only the columname in InfinispanEmbedded
+
+==== MongoDB
+
+==== Neo4j
+
+==== Integration Test and Provisioning
```

Comment:
I don't think we need this paragraph. The idea is to be synthetic here and provide information useful for the users. This is internal work.